ciao-launcher/main.go

```diff
@@ -341,6 +343,54 @@ func printClusterConfig() {
 	glog.Infof("Memory Limit:         %v", memLimit)
 }
 
+func discoverCertificate(certPath, certPrefix string) (string, error) {
+	ciaoCertsPath := "/etc/pki/ciao/"
+	fmt.Println("discoverCertificate")
+	if _, err := os.Stat(certPath); os.IsNotExist(err) {
+		files, err := ioutil.ReadDir(ciaoCertsPath)
+		if err != nil {
+			return certPath, err
```

Comment:
may make sense to return "" string here vs the certpath in case of error... I know user should always check error but sending back a valid string in the case of error may not make sense